SECURITY.md

```diff
@@ -0,0 +1,43 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `standard`
+project.
+
+  * [Reporting a Bug](#reporting-a-bug)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The `standard` team and community take all security bugs in `Open Build Service` seriously.
+Thank you for improving the security of `Open Build Service`. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the [Security Team](https://www.suse.com/de-de/support/security/contact/) at security@suse.com.
+
+The security team will acknowledge your email as soon as possible, and will send a
+more detailed response indicating the next steps in handling
+your report. After the initial reply to your report, the security team will
+endeavor to keep you informed of the progress towards a fix and full
+announcement, and may ask for additional information or guidance.
+
+Report security bugs in third-party modules to the person or team maintaining
```

Comment:
What are third-party modules? I guess this would warrant some examples..

Reply:
I know that. People don't, that's why we should include examples. Like:

Report security bugs in third-party modules (perl modules, ruby gems, used libraries) to the person or team maintaining them (upstream), not use please.